go-controller/pkg/ovn/ovn.go

```diff
@@ -453,6 +457,19 @@ func (oc *Controller) determineOVNTopoVersionFromOVN() (int, error) {
 	return ver, nil
 }
 
+func (oc *Controller) findLoadBalancerGroup() ([]nbdb.LoadBalancerGroup, error) {
```

Comment:
in libovsdbops you have findLBGroup...similar function name. I think this should be more specific? findClusterLBGroup?

Reply:
this function may not be necessary

Reply:
I removed it, it's not necessary indeed.